scripts/ingests/utils.py

```diff
@@ -721,26 +741,57 @@ def ingest_parallaxes(db, sources, plxs, plx_errs, plx_refs, save_db=False, verb
 
     n_added = 0
 
-    for i, source in enumerate(sources):
+    for i, source in enumerate(sources): # loop through sources with parallax data to ingest
         db_name = db.search_object(source, output_table='Sources')[0]['source']
 
-        # Search for existing parallax data.
+        # Search for existing parallax data and determine if this is the best
         # If no previous measurement exists, set the new one to the Adopted measurement
-        # TODO: figure out adopted in cases of multiple measurements.
         adopted = None
+        duplicate = False
         source_plx_data = db.query(db.Parallaxes).filter(db.Parallaxes.c.source == db_name).table()
```

Comment:
You can do something like: `db.search_object(source, output_table='Parallaxes')` to return the Parallaxes table for the matched source without having save a `db_name` variable that you use in the subsequent query. You still have access to the source name via the _source_ column in Parallaxes.

Reply:
Actually, I guess you may need to keep `db_name` for ingesting brand new parallaxes so maybe this isn't as useful.